docs/sources/articles/configuring.md

```diff
@@ -42,12 +58,18 @@ These options :
 
 ## Ubuntu
 
-After successfully [installing Docker for Ubuntu](/installation/ubuntulinux/), you can check the
-running status using Upstart in this way:
+As of `14.04`, Ubuntu uses `Upstart` as a process manager. By default, `Upstart` jobs
```

Comment:
Let me complete this article first and I can add a >= 15.04 section in a
new PR
ᐧ

On Tue, May 5, 2015 at 10:36 PM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In docs/sources/articles/configuring.md
> <https://github.com/docker/docker/pull/12804#discussion_r29709921>:
>
> > @@ -42,12 +58,18 @@ These options :
> >
> >  ## Ubuntu
> >
> > -After successfully [installing Docker for Ubuntu](/installation/ubuntulinux/), you can check the
> > -running status using Upstart in this way:
> > +As of `14.04`, Ubuntu uses `Upstart` as a process manager. By default, `Upstart` jobs
>
> I think the ubuntu section will have to be split into <= 14.10 and >=
> 15.04. Afaik, Ubuntu 15.04 switched to systemd, which results in people
> reporting that their configuration is no longer used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12804/files#r29709921>.
>



-- 
Ankush Agarwal
http://ankush.io
@ankushio <https://twitter.com/ankushio>

*There are 10 types of people in this world. Those who understand binary
and those who don't.*